XenAdmin/Wizards/NewNetworkWizard_Pages/NetWTypeSelect.cs

```diff
@@ -71,7 +71,9 @@ public NetworkTypes SelectedNetworkType
                                  ? NetworkTypes.External
                                  : rbtnCHIN.Checked
                                        ? NetworkTypes.CHIN
-                                       : NetworkTypes.Internal;
+                                       : rbtnSriov.Checked
+                                           ? NetworkTypes.SRIOV

```

Comment:
The design mentions that this checkbox may be disabled if there's no NIC supporting SR-IOV, I don't see an implementation for this.

Reply:
Yes, because this ticket is just about the implemention about the GUI. The XAPI part hasn't been ready, so I will intergrate it later. Thanks.